plugins/modules/aci_bd.py

```diff
@@ -458,22 +595,55 @@ def main():
             unkMacUcastAct=l2_unknown_unicast,
             unkMcastAct=l3_unknown_multicast,
             nameAlias=name_alias,
+            enableRogueExceptMac=enable_rogue_except_mac,
+            hostBasedRouting=host_based_routing,
+            intersiteBumTrafficAllow=allow_intersite_bum_traffic,
+            intersiteL2Stretch=allow_intersite_l2_stretch,
+            ipv6McastAllow=allow_ipv6_multicast,
+            llAddr=link_local_address,
+            mcastARPDrop=multicast_arp_drop,
+            vmac=vmac,
+            OptimizeWanBandwidth=optimize_wan_bandwidth,
         )
 
         if ipv6_l3_unknown_multicast is not None:
             class_config["v6unkMcastAct"] = ipv6_l3_unknown_multicast
 
-        aci.payload(
-            aci_class="fvBD",
-            class_config=class_config,
-            child_configs=[
-                {"fvRsCtx": {"attributes": {"tnFvCtxName": vrf}}},
-                {"fvRsIgmpsn": {"attributes": {"tnIgmpSnoopPolName": igmp_snoop_policy}}},
-                {"fvRsBDToNdP": {"attributes": {"tnNdIfPolName": ipv6_nd_policy}}},
-                {"fvRsBdToEpRet": {"attributes": {"resolveAct": endpoint_retention_action, "tnFvEpRetPolName": endpoint_retention_policy}}},
-                {"fvRsBDToProfile": {"attributes": {"tnL3extOutName": route_profile_l3out, "tnRtctrlProfileName": route_profile}}},
-            ],
-        )
+        child_configs = [
```

Comment:
Should we add to documentation how we can remove these named children? I think passing in a "" should be suffice to remove the formed relation but this would need to be tested.

Reply:
Yep, an empty string was enough. I have added extra test cases and example in the documentation.

Reply:
Can you also add to the documentation for those attributes where it applies: `- To delete this attribute, pass an empty string.` or something similar. This is what @gmicol introduced in other modules if not mistaken.

Reply:
Done